packages/esm-appointments-app/src/patient-chart/patient-appointments-cancel-modal.component.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
```

Comment:
As an side, this model seems something that is ripe for reuse, but not tackling that now.